api/swagger.yaml

```
@@ -9231,12 +9231,24 @@ paths:
       parameters:
         - name: "name"
           in: "path"
-          description: "Image name or ID."
+          description: |
+            Name of the image to push, for example, `registry.example.com/myimage`.
+            The image must be present in the local image store and tagged with
+            the given name.
```

maybe avoid "tagged" with here? 😅 

```suggestion
            The image must be present in the local image store with the same name.
```